docs/dasharo-tools-suite/documentation.md

```diff
@@ -5,18 +5,18 @@
 Dasharo Tools Suite was prepared to run on x86 platforms, but we can confirm
 that it boots on the following platforms:
 
-* ASUS KGPE-D16
-* Dell OptiPlex 7010/9010
+* ASUS KGPE-D16,
+* Dell OptiPlex 7010/9010,
 * MSI PRO Z690-A DDR4 ([test
-  report](https://docs.google.com/spreadsheets/d/16wokQYhtS7XA1DQC3Om7FY-IImG6SZisGK7NnzyRGVY/edit#gid=0&range=A75))
+  report](https://docs.google.com/spreadsheets/d/16wokQYhtS7XA1DQC3Om7FY-IImG6SZisGK7NnzyRGVY/edit#gid=0&range=A75)),
 * NovaCustom NV4x ([test
-  report](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=536764189&range=A161))
+  report](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=536764189&range=A161)),
 * NovaCustom NS5x/7x ([test
-  report](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=38447675&range=A174))
-* Protectli FW6/VP46xx
+  report](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=38447675&range=A174)),
+* Protectli FW6/VP46xx.
 
 <span style="color:red">
-Please do not use DTS to deploy Dasharo on MSI PRO Z690-A (WIFI) DDR5;
+Please do not use DTS v1.1.0 to deploy Dasharo on MSI PRO Z690-A (WIFI) DDR5;
 otherwise, you will brick your hardware. There are at least two bugs that have
 to be fixed before it is possible:
 
```

Comment:
What exactly does the following section refer to? Is this known issues section? If so, we should mark it up in some way.

Reply:
It refers to one user who did brick his DDR5 platform using DTS v1.1.0 for initial deploy. Note was added by @pietrushnic and I think that for now it should be still there as some may still use DTS v1.1.0

Reply:
@TomaszAIR OK, thanks for the clarification. Can't we somehow make this section more visible?